internal/control-panel/registry.lua

```diff
@@ -125,6 +125,8 @@ COMMANDS_BY_IDX = {
     {command='partial-items', help_command='tweak', group='gameplay', mode='tweak', default=true,
         desc='Displays percentages on partially-consumed items like hospital cloth.'},
     {command='pop-control', group='gameplay', mode='enable'},
+    {command='realistic-melting', help_command='tweak', group='gameplay', mode='tweak', default=false,
+        desc='Adjust selected item types melt return for all metals to ~95% of forging cost. Reduce melt return by 10% per wear level. Affects weapons, shields, armor parts, tools, and trap components.'},
```

Comment:
this is pretty long -- could you open `gui/control-panel` and make sure this fits in the help area without scrolling?

Reply:
Does not fit without scrolling, Sentence with "Affects weapons, shields, armor parts, tools, and trap components." has to be scrolled to.
![obraz](https://github.com/user-attachments/assets/ae902574-65d2-41bf-bd9f-fbcbdb94cf73)
![obraz](https://github.com/user-attachments/assets/92d6c99c-54d5-40b7-80d0-ff3601811538)

Reply:
see if you can trim the description down so it fits in the two-line space at the default (minimum) `gui/control-panel` window size

otherwise, scroll keys start acting funny

Reply:
Ok, removed last sentence.